templates/single-product/short-description.php

```diff
@@ -29,5 +29,5 @@
 
 ?>
 <div class="woocommerce-product-details__short-description">
-	<?php echo $short_description; // WPCS: XSS ok. ?>
+	<?php echo wp_kses_post($short_description); // WPCS: XSS ok. ?>
```

Comment:
There's multiple things that gets applied to `woocommerce_short_description` filter, so applying `wp_kses_posts()` so late like that will cause issues for many people.